sql/core/src/main/scala/org/apache/spark/sql/execution/LocalTableScanExec.scala

```diff
@@ -48,6 +48,14 @@ private[sql] case class LocalTableScanExec(
     }
   }
 
+  override protected def stringArgs: Iterator[Any] = {
+    if (rows.isEmpty) {
+      Iterator("Empty", output)
```

Comment:
Just to make sure, do you prefer the following output?
```
LocalTableScan EmptyRelation, [_1#58,_2#59,_3#60]
```

Instead of 
```
LocalTableScan Empty, [_1#58,_2#59,_3#60]
```